programs/fall-workshops.md

```diff
@@ -47,7 +49,7 @@ PiE on the Road will be held at the following locations and times.
         <p>Location:<br>
```

Comment:
Lastly, for all the "Time: ", "Location: ", and Deadline text, can you bold them? (Use the `<strong></strong>` tags)